frontend/packages/ceph-storage-plugin/src/utils/alert-action-path.tsx

```diff
@@ -1,13 +1,9 @@
-import { Alert } from '@console/internal/components/monitoring/types';
-import { history } from '@console/internal/components/utils';
 import { k8sGet } from '@console/internal/module/k8s';
 import { CEPH_STORAGE_NAMESPACE } from '../constants';
 import { OCSServiceModel } from '../models';
 
-export const getAlertActionPath = (alertData: Alert) =>
-  history.push(
-    `/k8s/cluster/nodes/${alertData.labels.host}/disks?node-disk-name=${alertData.labels.device}`,
-  );
+export const getDiskAlertActionPath = () =>
+  window.open('https://access.redhat.com/solutions/5194851');
```

Comment:
Please make sure the edit Doc on access.redhat.com/solutions won't change this URL

Reply:
This is KCS article it wont change

Reply:
ack

Reply:
Since this is an External Link I am not sure we would want to show it as part of the alert. If we want to shouldn't this show the External Link icon that is used for such links. 
cc @yuvalgalanti

Reply:
I guess this is a function `getDiskAlertActionPath` @bipuladh , we wont be able to do that .
Your point makes sense but wont we need changes in extension for this  ??